cohorts/2024/01-docker-terraform/homework.md

```diff
@@ -66,11 +66,13 @@ Remember that `lpep_pickup_datetime` and `lpep_dropoff_datetime` columns are in
 - 15859
 - 89009
 
-## Question 4. Largest trip for each day
+## Question 4. Longest trip for each day
```

Comment:
Might be even clearer if this said "Day with the longest single trip"

Reply:
I was just trying my best to avoid the word "maximum", but in this case it might not be a bad idea. It all depends if you are testing SQL skills or gathering requirements and data literacy.